src/commands/templates/install.js

```diff
@@ -15,13 +15,37 @@ const { runScript } = require('../../lib/helper')
 const { writeObjectToPackageJson, readPackageJson, getNpmDependency, processNpmPackageSpec, TEMPLATE_PACKAGE_JSON_KEY } = require('../../lib/npm-helper')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app-templates:templates:install', { provider: 'debug' })
 
+// aio-lib-console-project-installation depenencies
+const path = require('path')
+const loadConfig = require('@adobe/aio-cli-lib-app-config')
+const templateHandler = require('@adobe/aio-lib-console-project-installation')
+
 class InstallCommand extends BaseCommand {
   async run () {
     const { args } = await this.parse(InstallCommand)
     let templateName
 
     await runScript('npm', process.cwd(), ['install', args.path])
 
+    // Setup Developer Console App Builder project from template install.yml configuration file
+    // 1. Get an API access token from the developer console & install.yml file path
+    await this.login()
+
+    const installConfigFile = path.join(
+      process.cwd(),
+      'node_modules',
+      args.path,
```

Comment:
@roribio , it might take the following values https://github.com/adobe/aio-cli-plugin-app-templates/blob/55a3a71deab6afb78798a3e7f049ee7c30219284/src/commands/templates/install.js#L52-L65
you need to use `templateName` set in https://github.com/adobe/aio-cli-plugin-app-templates/blob/55a3a71deab6afb78798a3e7f049ee7c30219284/src/commands/templates/install.js#L29-L34

Reply:
Fixed ✅